product_supplierinfo_for_customer_elaboration/tests/test_customerinfo_elaboration.py

```diff
@@ -24,7 +24,7 @@ def setUpClass(cls):
             {
                 "name": cls.customer0.id,
                 "product_tmpl_id": cls.product.product_tmpl_id.id,
-                "elaboration_id": cls.elaboration.id,
+                "elaboration_ids": cls.elaboration,
```

Comment:
I think this is working by chance.

```suggestion
                "elaboration_ids": [(4, cls.elaboration.id)],
```